pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/signed-exchange.md

```diff
@@ -222,11 +179,10 @@ DevTools will show a `200` with a `signed-exchange` row, and a `from signed-exch
 
```

Comment:
cl/245998140 just landed in prod, so you can document the `warning` header on SXG failures ([e.g.](https://github.com/ampproject/amppackager/pull/304)). Probably worth disclaiming that work is ongoing to make the warning messages less cryptic.

Reply:
Added "A `warning` header may also be present that provides additional information."